android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt

```diff
@@ -499,6 +541,11 @@ class RNCWebViewManagerImpl {
         view.setMessagingEnabled(value)
     }
 
+  fun setBlobDownloadingEnabled(view: RNCWebView, value: Boolean) {
+    view.setDownloadingBlobEnabled(value)
+    view.setDownloadingMessage(getDownloadingMessageOrDefault())
```

Comment:
It seems like you wanted a "downloaded" message, not the "downloading" message that `getDownloadingMessageOrDefault` is tied to...?